03-loops.md

```diff
@@ -412,8 +412,7 @@ for idx = 1:3
 end
 ~~~
 
-Remember that to run our script, we simply type in its name in the
-command line:
+Save the `analyze.m` script as a new script called `analyze_loops.m`. To do this, we go to the `Editor` tab, click on the arrow below `Save`, and then on `Save As...`. Remember that to run our script, we simply type in its name in the command line:
```

Comment:
Thanks @gcapes and @shwina for your feedback. 

@gcapes:

re your first comment: 

I agree that the first `assert` needs to go before the `x0`, `y0`, `x1`, `y1` assignments. I'll make that change but leave the other two `assert` statements where they are.

re your second comment: 

I initialise `num` again to make it consistent with the two scripts above that one and to make it consistent with the python lesson (See http://swcarpentry.github.io/python-novice-inflammation/05-cond.html). I'm happy to drop the initialisation though, if you think it results in a better example. 

@shwina:

I will split the line using the semantic linefeeds approach.